src/service/protected_area.js

```diff
@@ -223,6 +223,28 @@ module.exports = (paPersistence, seService) => {
      * @return {Object} Geojson object with the geometry
      */
     getSELayer: async (categoryName, seType) => seService.getSELayerInPA(categoryName, seType),
+
+    /**
+     * Get the current human footprint layer divided by categories in a given
+     * environmental authority
+     * @param {String} categoryName protected area category
+     *
+     * @return {Object} Geojson object with the geometry
+     */
+    getHFCategoriesLayerById: async (categoryName) => {
```

Comment:
maybe this should be called "...ByCategory"?

Reply:
Done